readme.md

```diff
@@ -755,6 +755,7 @@
 - [Engineering Team Management](https://github.com/kdeldycke/awesome-engineering-team-management#readme) - How to transition from software development to engineering management.
 - [Developer-First Products](https://github.com/agamm/awesome-developer-first#readme) - Products that target developers as the user.
 - [Billing](https://github.com/kdeldycke/awesome-billing#readme) - Payments, invoicing, pricing, accounting, marketplace, fraud, and business intelligence.
+- [Engineering Strategy](https://github.com/aleixmorgadas/awesome-engineering-strategy#readme) - Books, posts, templates, learning materials, and open engineering strategies made by the community and companies.
```

Comment:
Per guidelines, the description should describe the topic, not the list itself.

Reply:
I have updated the description 👍